game_patch/os/commands.cpp

```diff
@@ -92,6 +92,34 @@ DcCommandAlias map_info_cmd{
     level_info_cmd,
 };
 
+ConsoleCommand2 server_rcon_password_cmd{
+    "server_rcon_password",
+    [](std::optional<std::string> new_rcon_password) {
+        if (!rf::is_multi || !rf::is_dedicated_server) {
+            rf::console::print("This command can only be run on a dedicated server!");
+            return;
+        }
+
+        if (new_rcon_password) {
+            if (new_rcon_password->size() > 16) {
+                // game limits client requests to 16 characters
```

Comment:
this comment is a bit wrong, the password is limited, not a "request" whatever that is

Reply:
I used the term "request" because the command is `rcon_request`

But I can change the comment.